actors/common/src/main/java/org/sunbird/learner/actors/user/service/UserService.java

```diff
@@ -0,0 +1,77 @@
+package org.sunbird.learner.actors.user.service;
+
+import java.util.List;
+import java.util.Map;
+
+import org.apache.commons.lang3.StringUtils;
+import org.sunbird.cassandra.CassandraOperation;
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.response.Response;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.models.util.ProjectLogger;
+import org.sunbird.common.models.util.datasecurity.EncryptionService;
+import org.sunbird.common.responsecode.ResponseCode;
+import org.sunbird.helper.ServiceFactory;
+import org.sunbird.learner.util.DataCacheHandler;
+import org.sunbird.learner.util.Util;
+import org.sunbird.learner.util.Util.DbInfo;
+
+public class UserService {
+	
+	private CassandraOperation cassandraOperation = ServiceFactory.getInstance();
+	  private EncryptionService encryptionService =
+	      org.sunbird.common.models.util.datasecurity.impl.ServiceFactory.getEncryptionServiceInstance(
+	          null);
+	  private DbInfo userDb = Util.dbInfoMap.get(JsonKey.USER_DB);
+
+	@SuppressWarnings("unchecked")
+	  public void checkPhoneUniqueness(String phone) {
+	    // Get Phone configuration if not found , by default phone will be unique across
+	    // the application
+	    String phoneSetting = DataCacheHandler.getConfigSettings().get(JsonKey.PHONE_UNIQUE);

```

Comment:
We can assume that phone and email is unique within our system for users. Therefore, we can remove the if checks on phone / email uniqueness.